source/agora/test/Quorum.d

```diff
@@ -54,52 +54,26 @@ unittest
     auto nodes = network.clients;
     network.expectBlock(Height(8), 5.seconds);
 
-    Transaction makePayTx (in Transaction prev, PublicKey[] addresses, uint index = 0)
-    {
-        auto input = Input(hashFull(prev), index);
-
-        Transaction tx =
-        {
-            TxType.Payment,
-            [input],
-            addresses.map!(addr => Output(Amount.MinFreezeAmount, addr)).array
-        };
-
-        auto signature = WK.Keys.Genesis.secret.sign(hashFull(tx)[]);
-        tx.inputs[0].signature = signature;
-        return tx;
-    }
-
-    Transaction makeFreezeTransaction (in Transaction prev, PublicKey address)
-    {
-        auto input = Input(hashFull(prev), 0);
-
-        Transaction tx =
-        {
-            TxType.Freeze,
-            [input],
-            [Output(Amount.MinFreezeAmount, address)]
-        };
-
-        auto signature = WK.Keys.Genesis.secret.sign(hashFull(tx)[]);
-        tx.inputs[0].signature = signature;
-        return tx;
-    }
+    KeyPair[] genesisKeyPair = [WK.Keys.Genesis];
```

Comment:
Why this ?

Reply:
`split()` takes a range of KeyPair, but I only need the genesis `KeyPair`. Instead of mapping the `WK.Keys.byRange`, I am using it like this: `.split(genesisKeyPair[0].address.repeat(3))`
Do you have a suggestion on how to structure this?

Reply:
A range of `PublicKey`, no ? And you're using `repeat` already. `only` would also work for a single element.